### PR TITLE
Added the countries_trivia script

### LINKED ISSUE #10: countries_trivia_game_script_load_questions_from_csv
## Script Title - countries_trivia.py

**what will change** -

## Instructions

- Create a new folder for your script and file/folder name should be appropriate.
- Create a README.md in your folder for program Instructions
- add requirements.txt if needed

Please add/delete options that are not relevant.

- [x] Adding New Code
- [ ] Improving Code
- [ ] Improving Documentation
- [ ] Bug Fix

### Programming Language

- [x] Python

:star2: Star it :fork_and_knife:Fork it :handshake: Contribute to it!

Discord server  - https://discord.gg/FXyh2S3

Happy Coding,

### PR DESCRIPTION
**Pull request by propenster**
### Self Check(Tick After Making pull Request)

- [] One Change in one Pull Request
- [X] This issue was assigned to me
- [X] My file is in proper folder
- [X] I am following clean code and Documentation.
- [X] I have added README.md with my script

### CODE REVIEW

**Comment by propenster** [comment]
@codePerfectPlus could you assign this to Arnab Sen? (I'm not sure of his Github username though) He said he'd like to do the task.

**Comment by codeperfectplus** [comment]
You already made a pull request for the same. I will assign him on a new issue. 
I will create some new issues in the evening.

**Comment by propenster** [comment]
Alright Boss.
Thanks.

File: countries_trivia_game/countries_trivia.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			"\n\n"
+			)
+
+def main():
+	score = 0
+	right = 0
+	wrong = 0
+	print()
+	instructions()
+	result = convert_csv_to_dict("C:\\Users\\Win8\\Desktop\\propenster\\java\\dev\\african_countries.csv")
```
Comment (codeperfectplus) [inline countries_trivia_game/countries_trivia.py:40]: Your game is good. but a little  correction in the path.
`convert_csv_to_dict("C:\\Users\\Win8\\Desktop\\propenster\\java\\dev\\african_countries.csv")`

change it to 
`convert_csv_to_dict("african_countries.csv")`

**Review by codeperfectplus — APPROVED** [review]
Thanks for Making Contributions.

**Pull request merged**